test/process_test.cc

```diff
@@ -245,7 +245,9 @@ TEST_P(ProcessTestWithSimTime, ScheduleAheadWorks) {
       ASSERT_EQ(output.results_size(), 1);
       EXPECT_EQ(Envoy::ProtobufUtil::TimeUtil::TimestampToNanoseconds(
                     output.results()[0].execution_start()),
-                options_->scheduled_start().value().time_since_epoch().count());
+                std::chrono::duration_cast<std::chrono::nanoseconds>(
+                    options_->scheduled_start().value().time_since_epoch())
```

Comment:
Ugh :/ this must have been annoying to figure out.. is this clang vs gcc?

Reply:
This one actually wasn't too bad!  Reading chrono duration documentation wasn't super fun though.

I assume this is clang vs gcc or something like that?  Honestly, I didn't figure this out.  It was broken on my computer but not in docker, so it's something like that, but I didn't investigate what the actual difference was... sorry.